src/app/Button/Button.js

```diff
@@ -1,37 +1,40 @@
-import styled from 'styled-components/macro'
 import PropTypes from 'prop-types'
+import styled from 'styled-components/macro'
 
-const ButtonStyled = styled.button`
-    padding: 7px;
-    display: flex;
-    align-items: center;
-    justify-content: space-evenly;
-    width: 80px;
-    height: 34px;
-    border-style: none;
-    border-radius: 5px;
-    box-shadow: 3px 3px 10px var(--primary-dark);
-    background-color: ${(props) => props.main ?
-        'var(--secondary-dark)' : 'var(--text-light)'};
-    color: ${(props) => props.main ?
-        'var(--text-light)' : 'var(--secondary-dark)'};
-    font-size: 1rem;
+const ButtonWrapper = styled.button`
```

Comment:
I think in this case `styled` was a better choice because it's the style of this button element itself, not an enclosing element like a `div`.